dockstore-integration-testing/src/test/java/io/dockstore/webservice/UserResourceIT.java

```diff
@@ -274,10 +280,21 @@ public void testSelfDestruct() throws ApiException {
             adminWorkflowsApi.getWorkflowByPath(WorkflowIT.DOCKSTORE_TEST_USER2_HELLO_DOCKSTORE_WORKFLOW, null, false);
 
         } catch (ApiException e) {
+            assertTrue(e.getCode() == 400);
             expectedAdminAccessToFail = true;
         }
         assertTrue(expectedAdminAccessToFail);
 
+        // Verify that self-destruct also deleted the service
+        boolean expectedAdminServiceAccessToFail = false;
+        try {
+            adminWorkflowsApi.getWorkflowByPath(SourceControl.GITHUB + "/" + serviceRepo, null, true);
+        } catch (ApiException e) {
+            assertTrue(e.getCode() == 400);
```

Comment:
Can use a constant to be more readable, something like the version 4 equivalent
https://hc.apache.org/httpclient-3.x/apidocs/org/apache/commons/httpclient/HttpStatus.html#SC_BAD_REQUEST